workflow/controller/operator_test.go

```diff
@@ -897,7 +897,7 @@ spec:
         arguments:
           parameters:
           - name: message
-            value: "{{item.os}} {{item.version}}"
+            value: "{{item.os}} {{item.version}} JSON({{item}})"
```

Comment:
Minor, but this test actually fails when run e2e because the `(` are sent unespaced to `cowsay`. 

Example:
```yaml
Name:                expand-with-items-whklw
Namespace:           argo
ServiceAccount:      default
Status:              Failed
Message:             child 'expand-with-items-whklw-312975041' failed
Created:             Mon Jan 06 07:19:39 -0800 (12 seconds ago)
Started:             Mon Jan 06 07:19:39 -0800 (12 seconds ago)
Finished:            Mon Jan 06 07:19:51 -0800 (now)
Duration:            12 seconds

STEP                                                  PODNAME                             DURATION  MESSAGE
 ✖ expand-with-items-whklw (expand-with-items)                                                      child 'expand-with-items-whklw-312975041' failed
 └-·-✖ whalesay(0:os:debian,version:9.1) (whalesay)   expand-with-items-whklw-312975041   11s       failed with exit code 2
   ├-✖ whalesay(1:os:debian,version:9.1) (whalesay)   expand-with-items-whklw-2232514396  5s        failed with exit code 2
   └-✖ whalesay(2:os:ubuntu,version:16.1) (whalesay)  expand-with-items-whklw-2191977801  7s        failed with exit code 2
$  pl expand-with-items-whklw-312975041
sh: 1: Syntax error: "(" unexpected
```

Could you add surrounding quotes to the arguments of `cowsay` below? E.g. `args: ["cowsay \"{{inputs.parameters.message}}\""]`

Reply:
Done.